### PR TITLE
Set content-length when known.

### PR DESCRIPTION
**Pull request by ZetaTwo**
If we can determine the length of the response, it should be set in the Content-Length header.

Now, it was some time ago I wrote this but I think that this fixed and issue with video/sound seeking in Firefox (or Chrome).

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -74,6 +74,10 @@ module.exports = function * (next) {
   } else {
     this.body = rawBody.slice.apply(rawBody, args);
   }
+
+  if(len !== '*') {
```
Comment (yorkie) [inline index.js]: `if (len !== '*')`

**Comment by ZetaTwo** [comment]
Absolutely. I'm a little new to PR. Do I need to approve that change somehow?

**Comment by yorkie** [comment]
@ZetaTwo You need to fix [this nit](https://github.com/koajs/koa-range/pull/9/files#r84203982)

**Comment by ZetaTwo** [comment]
Uhm, did I do this correctly now?

**Comment by yorkie** [comment]
Merged :)

**Pull request merged**